testbed/testbed/mock_backend.go

```diff
@@ -172,13 +173,20 @@ func (mb *MockBackend) ConsumeMetricOld(md consumerdata.MetricsData) {
 	}
 }
 
+type TraceDualConsumer interface {
+	consumer.TraceConsumer
+	consumer.TraceConsumerOld
+}
+
+var _ TraceDualConsumer = (*MockTraceConsumer)(nil)
+
 type MockTraceConsumer struct {
```

Comment:
Does this need to be public?

Reply:
Yes, it will be used by the test harness in the metrics package as a placeholder trace consumer.